tests/Integration/UI/SettingsPageTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Settings page tests.
+ *
+ * @package Parsely
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\Tests\Integration\UI;
+
+use Parsely\Parsely;
+use Parsely\Tests\Integration\TestCase;
+use Parsely\UI\Settings_Page;
+
+/**
+ * Row actions tests.
```

Comment:
I think this might be a leftover from copy pasting, but this tests are not testing row actions.

Reply:
This needs to be there (we use `self::$settings_page->validate_options`).

Reply:
I mean the `Row actions tests.` comment. It should be `Settings page tests.`